includes/class-rest.php

```
@@ -388,23 +388,26 @@ public function delete_optimised_image(): \WP_REST_Response {
 		public function import_settings( \WP_REST_Request $request ) {
```

PHPStan (2.1.46)

[error] Method PerformanceOptimise\Inc\Rest::import_settings() has invalid return type WP_REST_Response.


(class.notFound)